dataflow_exhaustive_test.py

```diff
@@ -0,0 +1,29 @@
+from pytest import mark
```

Comment:
How about calling the original test file `documentation_test.py` or `documenting_test.py`, and this this one ... well, *dataflow* is superfluous in the name, now that dataflow is all that there is. *Exhaustive* works. Maybe *robust*.

Reply:
Yes, I was thinking about renaming this test file, but I forgot to mention it yesterday. What do you prefer, a first commit changing the name of the file or a last commit renaming them?

Reply:
No strong opinion. As you know, I like a clean history, which leans me towards getting the name right in the first place. OTOH I don't want to mess with the historical commits that came from IC, so we're stuck with the wrong name at the beginning of history. At which point it really doesn't matter when the name changes. Do whatever is easiest: don't waste any time on Git-fiddling on this.